### PR TITLE
Adjust FE rules

### PR DESCRIPTION
**Pull request by muhammadimranfarooqi**

### CODE REVIEW

File: frontend/app_dbs_nossl.conf [hunk 1]
```diff
@@ -2,5 +2,9 @@ RewriteRule ^(/dbsproxy(/.*)?)$ https://%{SERVER_NAME}${escape:$1}%{env:CMS_QUER
 RewriteRule ^(/dbs2go(/.*)?)$ https://%{SERVER_NAME}${escape:$1}%{env:CMS_QUERY} [R=301,NE,L]
 RewriteRule ^(/dbs2go-writer(/.*)?)$ https://%{SERVER_NAME}${escape:$1}%{env:CMS_QUERY} [R=301,NE,L]
 RewriteRule ^(/dbs2go-migrate(/.*)?)$ https://%{SERVER_NAME}${escape:$1}%{env:CMS_QUERY} [R=301,NE,L]
+
+# this rule is for Python DBS server
```
Comment (vkuznet) [inline frontend/app_dbs_nossl.conf:6]: put here comment like:
```
# this rule is for accessing DBS server (either Python or GO based)
```

File: frontend/app_dbs_nossl.conf [hunk 2]
```diff
@@ -2,5 +2,9 @@ RewriteRule ^(/dbsproxy(/.*)?)$ https://%{SERVER_NAME}${escape:$1}%{env:CMS_QUER
...
 RewriteRule ^(/dbs(/.*)?)$ https://%{SERVER_NAME}${escape:$1}%{env:CMS_QUERY} [R=301,NE,L]
+
+# This rule is for GO DBS server
```
Comment (vkuznet) [inline frontend/app_dbs_nossl.conf:9]: and here put comment like:
```
# This rule is only for testing Go DBS server, it should be accompany with appropriate rules from app_dbs_ssl.conf
```

Comment (vkuznet) [reply frontend/app_dbs_nossl.conf:9]: and you'll need to comment out the actual rule since we no longer need /dbsnew end-point

**Review by vkuznet — CHANGES_REQUESTED** [review]

File: frontend/app_dbs_nossl.conf (hunk 2 above)
Comment (muhammadimranfarooqi) [reply frontend/app_dbs_nossl.conf]: `/dbsnew` end-point is still used for DBSWriter

**Pull request merged**